x/ibc/03-connection/keeper/handshake.go

```diff
@@ -226,31 +198,32 @@ func (k Keeper) ConnOpenConfirm(
 		return sdkerrors.Wrap(types.ErrConnectionNotFound, "cannot relay ACK of open attempt")
 	}
 
-	if connection.State != types.TRYOPEN {
+	if connection.State != exported.TRYOPEN {
 		return sdkerrors.Wrapf(
 			types.ErrInvalidConnectionState,
 			"connection state is not TRYOPEN (got %s)", connection.State.String(),
 		)
 	}
 
+	// NOTE: should be safe to use proofHeight here
+	// TODO: Update spec
```

Comment:
It's updated in https://github.com/cosmos/ics/pull/348 :smile:

Reply:
`ConnOpenConfirm` hasn't been updated yet